.travis.docker-push.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+version=`cat VERSION`
+images=(
+    "function-controller"
+    "function-sidecar"
+    "http-gateway"
+    "topic-controller"
+)
+
+docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD"
```

Comment:
Might be worth using the `--stdin` flag for this: https://docs.docker.com/engine/reference/commandline/login/#parent-command. They claim it'll prevent leaking to logs. That said I think travis blocks it too.

Reply:
This is the format travis uses in their docs, the warning is annoying, but `echo`ing the password doesn't feel better to me

Reply:
Works for me